kernel/src/main/java/org/kframework/compile/ExpandMacros.java

```diff
@@ -220,6 +220,9 @@ public K apply(KApply k) {
                                 public K apply(KVariable k) {
                                     K result = subst.get(k);
                                     if (result == null) {
+                                      if (k.name().equals("_Configuration")) {
```

Comment:
I'm not entirely sure what's going on here, but I don't like that we're calling out variables by name specifically. Would this be affected if I named a variable as `BLAH_Configuration` in my definition?

Reply:
No, it's an exact string match, and this particular string is not part of the syntax of variables in K, so it can't be written by the user. This isn't really a new design choice though; the function context pass was already written this way, we're just fixing a bug in how it interacts with macros.